Saule/Http/NoDefaultIncludedAttribute.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Web.Http.Controllers;
+using System.Web.Http.Filters;
+
+namespace Saule.Http
+{
+    /// <summary>
+    /// Indicates that the action should not include related data by default.
+    /// </summary>
+    [AttributeUsage(AttributeTargets.Method)]
+    public sealed class NoDefaultIncludedAttribute : ActionFilterAttribute
```

Comment:
Yeah, after thinking about this for a bit longer I think your initial idea was right. I think something like this should work:

| |`AllowsQuery` off |`AllowsQuery` on|
|---|---|---|
|**`NoDefaultIncluded` off** | Don't allow `include` query; always include all available relationships | When `include` provided, include only those relationships, otherwise everything |
|**`NoDefaultIncluded` on** | Don't allow `include` query; never include any available relationships | When `include` provided, include only those relationships, otherwise nothing |

The lower left here is a bit strange, and could be changed if that makes sense. I'm not a huge fan of the name `NoDefaultIncluded`, because all other attributes have an active verb (e.g. `ReturnsResource`, `AllowsQuery`). Unless you can think of something better I think it should be fine though :smile: